guides/parameters_binding.md

```diff
@@ -0,0 +1,52 @@
+# Parameters binding Guide
+
+This guide explains how the binding for request parameters and fields inside a parameter objects
+works and what happens when the binding fails.
+
+
+### Parameter Objects
+Lets have a look at this example:
+
+```java
+@Data
+@OpenApi.Shared
+class EntryQueryParams {
+    int page = 1;
+    int pageSize = 50;
+    String filter;
+    CustomField custom = CustomField.empty();
+}
+
+class EntryController {
+    @GetMapping( "/entries" )
+    List<Entry> queryEntries(EntryQueryParams params, @RequestParam boolean special) {
+        EntryQuery query = EntryQuery.from(params)
+            .withSpecial(special)
+            .withUser(currentUser);
+        return entryQueryService.runQuery(query);
+    }
+}
+```
+
+Spring will try to convert parameters from String into the proper type using a `PropertyEditor` if available, otherwise a `Converter`. `PropertyEditor`s are registered and bind to a class in `CrudControllerAdvice.initBinder(WebDataBinder binder)` method.
+`Converter`s are registered in `WebMvcConfig.addFormatters(FormatterRegistry registry)` method.
+
+`PropertyEditor`s and `Converter`s work in a similar way but in different context, the former is used only when binding parameters, the latter is a global converter that can be used in any layer of the system.
```

Comment:
Do you mean binding "web" parameters? As binding parameters is ambiguous to me. If that is true you could say that `PropertyEditor` are only applied in the Spring web MVC context to bind request parameters.